app/actions/action_user.js

```diff
@@ -8,11 +8,17 @@ export const logIn = (payload) => {
     payload
   };
 };
-
+export const updateFriends = (payload) => {
+  return{
+    type: UPDATE_FRIENDS,
+    payload
+  };
+};
 export const firebase_check = (userCredentials) => {
   let id = userCredentials.userId;
```

Comment:
The reassignment might be confusing, but with ES6 you can shorten this to:
`let { id: userId, token } = userCredentials`